src/widget/wsearchlineedit.cpp

```diff
@@ -82,11 +79,12 @@ WSearchLineEdit::WSearchLineEdit(QWidget* pParent)
       WBaseWidget(this),
       m_clearButton(new QToolButton(this)),
       m_foregroundColor(kDefaultForegroundColor),
-      m_state(State::InactivePlaceholder) {
+      m_state(State::Inactive) {
     DEBUG_ASSERT(kEmptySearch.isEmpty());
     DEBUG_ASSERT(!kEmptySearch.isNull());
 
     setAcceptDrops(false);
+    setPlaceholderText(tr("Search...", "noun"));
```

Comment:
```suggestion
    //: Placeholder text for empty library search bar
    setPlaceholderText(tr("Search...");
```

The second argument to `tr` is legacy. Adding a translator tip comment is preferred:
https://doc.qt.io/qt-5/i18n-source-translation.html#translator-comments
> The recommended way to add comments is to annotate the tr() calls in your code with comments

https://doc.qt.io/qt-5/i18n-source-translation.html#disambiguation

> In Qt 4.4 and earlier, this disambiguation parameter was the preferred way to specify comments to translators.